client/client.go

```diff
@@ -71,4 +71,12 @@ type Client interface {
 	UpdateUserActive(userId string, activate bool) (bool, *model.Response)
 	UpdateTeam(team *model.Team) (*model.Team, *model.Response)
 	UpdateChannelPrivacy(channelId string, privacy string) (*model.Channel, *model.Response)
+	CreateBot(bot *model.Bot) (*model.Bot, *model.Response)
+	// PatchBot(userId string, patch *model.BotPatch) (*model.Bot, *model.Response)
+	// GetBots(page, perPage int, etag string) ([]*model.Bot, *model.Response)
+	// GetBotsIncludeDeleted(page, perPage int, etag string) ([]*model.Bot, *model.Response)
+	// GetBotsOrphaned(page, perPage int, etag string) ([]*model.Bot, *model.Response)
+	// DisableBot(botUserId string) (*model.Bot, *model.Response)
+	// EnableBot(botUserId string) (*model.Bot, *model.Response)
+	// AssignBot(botUserId, newOwnerId string) (*model.Bot, *model.Response)
```

Comment:
With this API; `bot update`, `bot list` (with `--orphaned`, `--all` flags), `bot enable`, `bot disable` and `bot assign` subcommands can be added.